src/sql/workbench/parts/notebook/browser/notebook.css

```diff
@@ -136,4 +136,76 @@
 .notebookEditor .book-nav .dialog-message-button {
 	min-width: 100px;
 	margin-right: 10px;
-}
\ No newline at end of file
+}
+
+.notebookEditor .hoverButtonsContainer {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	width: 40%;
+	margin: 1px auto;
+}
+
+.notebookEditor .hoverButtonsContainer .containerBackground {
```

Comment:
I think this should also be invisible until we hover.

Reply:
Fixed